modules/gcloud/src/main/java/org/testcontainers/containers/DatastoreEmulatorContainer.java

```diff
@@ -0,0 +1,35 @@
+package org.testcontainers.containers;
+
+import java.util.Arrays;
+
+import org.testcontainers.containers.wait.strategy.Wait;
+
+/**
+ * A Datastore container that relies in google cloud sdk. The container provides
+ * additional instructions to install the components needed in the alpine images.
+ *
+ * Default port is 8081.
+ *
+ * @author Eddú Meléndez
+ */
+public class DatastoreEmulatorContainer extends GCloudGenericContainer<DatastoreEmulatorContainer> {
+
+	private static final String[] CMDS = {"apk --update add openjdk8-jre",
```

Comment:
Sorry for the mundane comment, but please could you reformat using 4 spaces for indentation? If you have an editorconfig-aware IDE this convention should get picked up automatically.